sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_vendor/storage/blob/_upload_helpers.py

```diff
@@ -147,8 +150,9 @@ def upload_block_blob(  # pylint: disable=too-many-locals
             validate_content=validate_content,
             headers=headers,
             tier=tier.value if tier else None,
+            blob_tags_string=blob_tags_string,
             **kwargs)
-    except StorageErrorException as error:
+    except HttpResponseError as error:
```

Comment:
Given that there's been some updates to error handling - do any of out live tests deal with errors raised from Storage? It shouldn't be an issue, but would be nice to know we have coverage :)

Reply:
nope, we don't have tests covering storage errors as for most blob storage errors, our sdk simply bubble up or ignore them.

The only errors we care about are `ResourceModifiedError, ResourceExistsError` which are treated as `OwnershipLostError`:
https://github.com/Azure/azure-sdk-for-python/blob/f0697ff8b1bcafbd1a82e7961098f14229295e2b/sdk/eventhub/azure-eventhub-checkpointstoreblob/azure/eventhub/extensions/checkpointstoreblob/_blobstoragecs.py#L163-L173


I don't think we need to cover all of the blob storage errors now based on our current "bubble up or ignore" approach. but I'm always open to adding new tests if needed :P